assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -280,11 +280,17 @@ che.infra.kubernetes.namespace.allow_user_defined=false
 #    and prepare workspace service account with needed roles there
 che.infra.kubernetes.service_account_name=NULL
 
+# DEPRECATED, use che.infra.kubernetes.workspace_sa_cluster_roles
 # Specifies an optional, additional cluster role to use with the workspace service account, to allow for addition
 # Note that the cluster role name must already exist, and the Che service account needs to be able to create a Role Binding
 # to associate this cluster role with the workspace service account.
 che.infra.kubernetes.cluster_role_name=NULL
 
+# Specifies optional, additional cluster roles to use with the workspace service account.
+# Note that the cluster role names must already exist, and the Che service account needs to be able to create a Role Binding
+# to associate these cluster roles with the workspace service account. The names are comma separated.
```

Comment:
we can add a notion about the old `che.infra.kubernetes.cluster_role_nam`  property here.

Reply:
done